doc/data-mining-library/source/reference/classification.rst

```diff
@@ -85,11 +85,6 @@ For data sets that include continuous attributes,
 .. index:: multilayer perceptron
    pair: classification; multilayer perceptron
 
-Multilayer perceptron (feed-forward neural network)
```

Comment:
The index above the removed section should be removed too. It is probably not used anywhere, but it wouldn't hurt to double check.